README.md

```diff
@@ -93,6 +93,56 @@ SELECT COUNT(jid) FROM sidekiq_jobs WHERE event = 'error' AND time > now() - 1d
 
 Et cetera.
 
+### Stats and Queues metrics
+
+To collect metrics for task stats and queues, you need to run the script periodically. For example you can use gem `clockwork`.
+
+For stats metrics:
+```ruby
+require "sidekiq/metrics/stats"
+
+Sidekiq::Metrics::Stats.new(
+            influxdb_client: InfluxDB::Client.new(options), # REQUIRED
+            series_name: 'sidekiq_stats',                   # optional, default shown
+            retention_policy: nil,                          # optional, default nil
+            tags: {},                                       # optional, default {}
+          ).call
+```
+
+For queues metrics:
+```ruby
+require "sidekiq/metrics/queues"
+
+Sidekiq::Metrics::Queues.new(
+            influxdb_client: InfluxDB::Client.new(options), # REQUIRED
+            series_name: 'sidekiq_queues',                   # optional, default shown
+            retention_policy: nil,                          # optional, default nil
+            tags: {},                                       # optional, default {}
+          ).call
+```
```

Comment:
I still don't understand: why there are two separate classes and why should the user call them both?

Why haven't you made one class that does all the work, so that the user has to add a single call to their clockwork configuration?

Reply:
So, what about if user needs only one metric? I think many class better then list init params and explicit is better than implicit. 

If you think that it would be better to make one class, then it is not difficult to rewrite.

Reply:
Alright, it's better to enable individual metrics.